src/app/Routes.tsx

```diff
@@ -223,6 +224,11 @@ const Routes: React.FC<RouteProps> = ({ alerts, currentUser }) => {
                     <Aside>
                       <UserAside fav="true" match={match} />
                     </Aside>
+                    <InitializationCard
```

Comment:
I think it would best for this to be nested within the `GlobalReadingsList` component. You'd be able to get rid of a lot of redundant logic that way.

Reply:
Do you mean add the Aside tag within the GlobalReadingList tag or combining the code from the UserAside.tsx file into the GlobalReadingList.tsx fille? When I nest the tag I get an error:

![image](https://user-images.githubusercontent.com/52865683/145734160-1b2bb8f5-acfe-41b3-a035-58a63371c8a6.png)
 
Sorry sent wrong screenshot. Here's the correct one:
![image](https://user-images.githubusercontent.com/52865683/145734256-b82be9a9-f2b1-4dfb-bce3-32d875918514.png)

![image](https://user-images.githubusercontent.com/52865683/145734293-6a5cf1d9-36a1-45d4-951a-c9c37ecdb9c2.png)

Reply:
Sorry for any confusion. The `InitializationCard` should live within the `GlobalReadingsList`. Right now it is is `Routes.tsx` lines 227-231, it should be moved withing `GlobalReadingsList.tsx`. I think this will also help with some of the css issues

Reply:
Gotcha. The issue I'm having here is the GlobalRedingsList.txt file has the VirtualizedList.tsx within it that controls the global readings. When I add the InitializationMessage.tsx here it overwrites the global readings for the entire site (controlled by VirtualizedList.tsx. Since I can only include one render I'm trying to figure out how to separate the two. Show global and show InitializationMessage when user clicks on "Your reads" to show their individual reads.

Reply:
This is an instance of poor component naming on my part. So the `GlobalReadingsList` is used by the "Global", "Your Reads", and "Friends" tabs. It's actually not just for the global list. It receives a `list` prop to determine which tab's readings to display. You could check if the `readings` prop is empty, and if it is and the `list` prop isn't "global", display the intializationMessage. Now that I'm thinking of it, the grammar in the message would need to work for an individual user's page, another user's page that I visit, and the "Friend's" tab in case a user is not following anyone